_includes/overview/provisioning.html

```diff
@@ -1,10 +1,5 @@
-<p>
-  For provisioning purposes you'll want to search for hosts that are unallocated. Do not allocate hosts in any other state.
-</p>
-<p>
-  To provision a host just click the 'Provision Server' link in the actions drop down. You will be presented with a dialog
-  like the following:
-</p>
+<p>For provisioning purposes you'll want to search for hosts that are unallocated (or sometimes maintenance), as they are not already claimed and in production. Do not provision hosts in any other state!</p>
+<p>To provision a host just click the 'Provision Server' link in the actions drop down. Assets in Allocated are not able to be provisioned until you move them into maintenance or unallocated. You will be presented with a dialog like the following:</p>
```

Comment:
I think the configuration page goes into more detail for provisioning. This page is meant to only give a terse description of whats possible, not get mired in configuration